oozie-to-airflow/tests/mappers/test_pig_mapper.py

```diff
@@ -120,7 +121,8 @@ def test_required_imports(self):
         ast.parse(imp_str)
 
     def test_first_task_id(self):
+        params = {"nameNode": "hdfs://"}
```

Comment:
It's an incorrect value. Can you add a hostname? We currently do not have validation, but it changes all the time.

Reply:
It's not incorrect. Most of our `job.properties` have this value. 
You can have a full path as e.g. `hdfs:///user/pig/examples/pig_test_node` and then I think it will look for the file/dir in the local hdfs (if you run it on a machine which has hdfs) instead of connecting to another machine.

If you look in the project for `hdfs:///` (three slashes) there's plenty of them. If we were to change it, we should change it for all in a separate PR, I think.

Anyway, here it just should be a non-empty value for the tests to pass, so it's not that important.

Reply:
hdfs:// is correct - it means "use the default cluster"

Reply:
I can not find it in the documentation. Can I ask for a link?